hr_expense_cancel/readme/USAGE.rst

```diff
@@ -1,14 +1,10 @@
 To use this module, you need to:
 
-#. Go to *Expenses -> My Expenses -> Expenses to Submit* and create a new
-   *Expense* with *Employee (to reimburse)* checked on the field *Payment By*
+#. Go to *Expenses -> My Expenses -> Expenses to Submit* and create a new *Expense* with *Employee (to reimburse)* checked on the field *Payment By*
```

Comment:
Don't change this if not needed

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)

Reply:
Fixed in [commit](https://github.com/OCA/hr/pull/564/commits/f4db176c4f844ed47892a5cb584695efc5573fc8)